lib/spack/spack/bootstrap.py

```diff
@@ -28,6 +31,216 @@
 import spack.util.path
 from spack.util.environment import EnvironmentModifications
 
+#: Map a bootstrapper type to the corresponding class
+_bootstrap_methods = {}
+
+
+def _bootstrapper(type):
+    """Decorator to register classes implementing bootstrapping
+    methods.
+
+    Args:
+        type (str): string identifying the class
+    """
+    def _register(cls):
+        _bootstrap_methods[type] = cls
+        return cls
+    return _register
+
+
+def _try_import_from_store(module, abstract_spec):
```

Comment:
```suggestion
def _try_import_from_store(module, abstract_spec_str):
```
maybe for consistency with the other methods in this module?

Reply:
Fixed in 234c8e9